src/database.tsx

```diff
@@ -1,16 +1,19 @@
 import { database } from 'firebase/app';
 import { list, object, QueryChange, listVal } from 'rxfire/database';
-import { ReactFireOptions, useObservable, checkIdField, checkStartWithValue } from '..';
+import { ReactFireOptions, useObservable, checkIdField, ObservableStatus } from './';
 
 import { Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
 
 const CACHED_QUERIES = '_reactFireDatabaseCachedQueries';
 
 // Since we're side-effect free, we need to ensure our observableId cache is global
+// @ts-ignore: TS doesn't like globalThis
```

Comment:
globalThis should have come with typescript 3.4 https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html, is there somethin missing in the tsconfig?

Reply:
I get the error:

```
Property '_reactFireDatabaseCachedQueries' does not exist on type 'typeof globalThis'
```

Maybe I need to find a way to declare the keys we expect to be on `globalThis`?

Reply:
Resolving since we figured this out together. We now have a `ReactFireGlobals` type and we just cast `globalThis`:

```ts
((globalThis as any) as ReactFireGlobals)
```